examples/table_of_integrals.py

```diff
@@ -49,8 +49,7 @@
 fl.next('frq-noise density')
 fl.plot(frq_noise)
 frq_noise.ift('temp')
-if frq_noise.get_ft_prop('temp','df') is not None:
-    frq_noise /= sqrt(ndshape(frq_noise)['temp']) * frq_noise.get_ft_prop('temp','df') # normalization
+frq_noise /= sqrt(ndshape(frq_noise)['temp']) * frq_noise.get_ft_prop('temp','df') # normalization
```

Comment:
this requires the most recent version of pyspecdata (if you're using a branch, merge master into it)

Reply:
it's not optional

Reply:
I merged in the master of pyspecdata but I am still getting an error 
`Traceback (most recent call last):
  File "C:\Users\Alex Guinness\gitstuff\proc_scripts\examples\table_of_integrals.py", line 52, in <module>
    frq_noise /= sqrt(ndshape(frq_noise)['temp']) * frq_noise.get_ft_prop('temp','df') # normalization
TypeError: unsupported operand type(s) for *: 'float' and 'NoneType'` I'm surprised you haven't run into this error

Reply:
it wasn't pushed, sorry!